dna-src/zomes/graphql/code/src/schema/post.rs

```diff
@@ -63,37 +104,49 @@ graphql_object!(Post: Context |&self| {
   	}
 
 	field comments(first: Option<i32>, cursor: Option<ID>, order: Option<String>) -> FieldResult<CommentQuerySet> {
+		let comments = self.get_comments()?;
+	    Ok(CommentQuerySet{
+	    	total: comments.len() as i32,
```

Comment:
This will definitely do the job for now. Formally this should be the number of agents who have commented though, not the total number of comments. If that's easy to achieve it'd be good to see how we collect that.